CMakeLists.txt

```diff
@@ -2,101 +2,35 @@ cmake_minimum_required(VERSION 2.6)
 
 project(FreeGish C)
 
-set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake-extras/;${CMAKE_MODULE_PATH}")
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake-extras)
 
 option(INSTALL_FHS "Install into Linux FHS" OFF)
+set(OpenGL_GL_PREFERENCE LEGACY)
 
-#don't pollute source dir
-if(NOT ${INSTALL_FHS})
-    set(EXECUTABLE_OUTPUT_PATH "${PROJECT_SOURCE_DIR}")
-endif(NOT ${INSTALL_FHS})
+include(FindPkgConfig)
 
-# Uncomment this to hardcode the data path. On the command line, you
-# would actually give -DDATAPATH='"/usr/share/games/gish"' or similar
-# but CMake does the extra quoting for us.
-#add_definitions(-DDATAPATH="/usr/share/games/gish")
+find_package(SDL2 REQUIRED)
+find_package(OpenAL REQUIRED)
+find_package(OpenGL REQUIRED)
+find_package(Vorbis REQUIRED)
+find_package(PNG REQUIRED)
+find_library(LIBM m)
 
 if(MSVC)
-    find_package(Ogg CONFIG REQUIRED)
-    find_package(Vorbis CONFIG REQUIRED)
-    find_package(libpng CONFIG REQUIRED)
-    find_package(SDL2 CONFIG REQUIRED)
-    find_package(OpenAL CONFIG REQUIRED)
-    find_package(OpenGL REQUIRED)
-
-else()
-    find_package(SDL2 REQUIRED sdl2)
-    find_package(OpenAL REQUIRED)
-    find_package(OpenGL REQUIRED)
-    find_package(Vorbis REQUIRED)
-    find_package(PNG REQUIRED)
-    find_library(M_LIBRARIES m)
+    set(EXECUTABLE_OUTPUT_PATH "${PROJECT_SOURCE_DIR}")
 endif()
 
+set(FREEGISH_INSTALL_DATA_DIR, ${CMAKE_INSTALL_DATADIR}/freegish)
 
+install(DIRECTORY animation DESTINATION ${FREEGISH_INSTALL_DATA_DIR}/)
```

Comment:
For me it tries to install this to `//animation` on Ubuntu. I think `CMAKE_INSTALL_DATADIR` is not set for me. This is problematic, though. This probably applies to all install statements right now. This is without using `INSTALL_FHS`, though.

Reply:
Indeed, this was broken. It had something to do with setting `FREEGISH_INSTALL_DATA_DIR`, which didn't work correctly. Looking at it again, I think it wasn't really needed to have a separate variable for it.

Also, the `DATADIR` define needed an absolute path, which it currently didn't get. Fixed now.